Apps/AdminWebClient/src/ClientApp/src/views/Communication.vue

```diff
@@ -53,6 +53,27 @@
         </v-col>
       </v-row>
     </v-form>
+
+    <v-row>
+      <v-col md="9">
+        <v-row>
+          <v-col no-gutters>
+            <v-data-table
+              :headers="tableHeaders"
+              :items="bannerList"
+              :items-per-page="5"
+            >
+              <template v-slot:item.effectiveOn="{ item }">
+                <span>{{ formatDate(item.effectiveDateTime) }}</span>
```

Comment:
Not 100% this one will work since it will be formatting the date before it can be sorted. It might sort on the date string representation rather than the date.

How about using a custom sort, something like: https://stackoverflow.com/questions/52678905/how-to-use-vutifys-custom-sort

Reply:
right, the format will not work here for a string formatted date time. we need to use a custom sort.